spec/models/trainee_spec.rb

```diff
@@ -23,5 +23,21 @@
         expect { trainee.dttp_id = uuid }.to raise_error(LockedAttributeError)
       end
     end
+
+    context "record set up validation" do
+      it "tests for the presence of a record type of new trainee" do
+        trainee = Trainee.new
```

Comment:
`subject` will be `Trainee.new` by default. So we could change this to...

```suggestion
  expect(subject).not_to be_valid
  expect(subject.errors.keys).to include(:record_type)
  
```